htdocs/web_portal/views/user/view_user.php

```diff
@@ -122,10 +122,12 @@
                     <tr class="site_table_row_2">
                         <td class="site_table">Home Site</td>
                         <td class="site_table">
-                            <a href="index.php?Page_Type=Site&amp;id=
-                                <?php echo $params['user']->getHomeSite()->getId()?>">
-                                <?php xecho($params['user']->getHomeSite()->getShortName()) ?>
-                            </a>
+                            <?php
+                            $homeId = $params['user']->getHomeSite()->getId();
+                            echo "<a href=\"index.php?Page_Type=Site&amp;id=$homeId\">";
+                            echo $params['user']->getHomeSite()->getShortName();
```

Comment:
As you're using a variable on line 127, I feel this part would be easier to read if it also used a variable that was set between 126 and 127.